pkgs/development/compilers/open-watcom/default.nix

```diff
@@ -0,0 +1,147 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+
+, withDocs ? false
+, dosbox
+, ghostscript
+, withGUI ? false
```

Comment:
Been afew months since I looked at these, I may have to go back and figure out which files they create, what their size & time impact is etc.

Reply:
Builds times & file differences:

- both disabled: ~28min
- withGui: ~32min, some additional GUI applications (only on DOS, Windows & OS/2)
- withDocs: ~84min, PDF documentation of the tools, APIs and introductions to C/C++, ~13MB
- both: ~170min(!), the above two differences + additional help files for GUI applications